app/models/guide.rb

```diff
@@ -53,6 +53,12 @@ def has_published_edition?
     editions.where(state: "published").any?
   end
 
+  def editions_since_last_published
+    latest_published_edition = editions.published.last
+    editions
+      .where("created_at > ?", latest_published_edition.created_at)
+  end
```

Comment:
I think that would make sense if we were guaranteed to always have editions in any context a guide was in. We're only guaranteed to not have editions when used in code path in question.

It reduces the reusability of the method.